aws/logs_monitoring/caching/common.py

```diff
@@ -3,40 +3,17 @@
 import logging
 import re
 from collections import defaultdict
-from telemetry import (
-    DD_FORWARDER_TELEMETRY_NAMESPACE_PREFIX,
-    get_forwarder_telemetry_tags,
-)
 
 logger = logging.getLogger()
 logger.setLevel(logging.getLevelName(os.environ.get("DD_LOG_LEVEL", "INFO").upper()))
 
-try:
```

Comment:
That `DD_SUBMIT_ENHANCED_METRICS` bit is important I think

Reply:
We were not using it in the cache metrics submission, only in enhanced metrics.

Reply:
In any case I added the same check here